mottak-arkiv-service/app/database/repositories/invitasjon_repository.py

```diff
@@ -16,6 +16,10 @@ def create(db: Session, arkivuttrekk_id: int, avgiver_epost: str, status: Invita
     return dbo
 
 
+def get_by_id(db: Session, id_: int) -> Invitasjon_DBO:
+    db.query(Invitasjon_DBO).get(id_)
```

Comment:
```suggestion
def get_by_id(db: Session, id_: int) -> Invitasjon_DBO:
    return db.query(Invitasjon_DBO).get(id_)
```

Burde ikke denne returnere noe?

Reply:
🙈  Jo